src/js/experimental/menu/media-settings-menu.js

```diff
@@ -0,0 +1,47 @@
+import { globalThis } from '../../utils/server-safe-globals.js';
+import { MediaChromeMenu } from './media-chrome-menu.js';
+import { getMediaController } from '../../utils/element-utils.js';
+
+const template = document.createElement('template');
+// prettier-ignore
+template.innerHTML = MediaChromeMenu.template.innerHTML + /*html*/`
+  <style>
+    :host {
+      ${/* Bottom fix setting menu items for animation when the height expands. */ ''}
+      justify-content: var(--media-settings-menu-justify-content, flex-end);
+      border-radius: 2px 2px 0 0;
+      overflow: hidden;
+    }
+
+    slot:not([name]) {
+      overflow: visible;
+    }
+
+    #container.has-expanded {
+      --media-settings-menu-item-opacity: 0;
+    }
+  </style>
+`;
+
+/**
+ * @cssproperty --media-settings-menu-justify-content - `justify-content` of the menu.
+ */
+class MediaSettingsMenu extends MediaChromeMenu {
+  static template = template;
+
+  /**
+   * Returns the anchor element when it is a floating menu.
+   * @return {HTMLElement}
+   */
+  get anchorElement() {
+    if (this.anchor != undefined) return super.anchorElement;
+    return getMediaController(this).querySelector('media-settings-menu-button');
```

Comment:
Should we make anchor always manual rather than automatic? It's not obvious to me how we'd turn this off in the apple tv player-takeover style, where we don't want anchoring to happen.

Reply:
fair point, I'll add that example back in. I removed that layout for a bit to get the more common use case as quickly as possible ready for review. Removing the anchoring right now would be setting `anchor=""` an empty string but it could also be `noanchor` for example. 

I can see the case for opt-in as well, it would be closer to how that web API would work. 
I'll try out how that looks in the HTML...

Reply:
Yeah, I think we definitely need something more clear than empty string. What about `anchor="auto"` or something like that, keeping the magic but making it explicit.

Reply:
I made this change, `anchor="auto"` is now required for auto anchoring...

also added the nested column menu 
https://media-chrome-git-fork-luwes-settings-menu-mux.vercel.app/examples/vanilla/control-elements/media-settings-menu.html